runtime/bundles/org.eclipse.core.runtime/src/org/eclipse/core/runtime/ILog.java

```diff
@@ -132,4 +134,29 @@ default void error(String message) {
 	default void error(String message, Throwable throwable) {
 		log(new Status(IStatus.ERROR, getBundle().getSymbolicName(), message, throwable));
 	}
+
+	/**
+	 * Returns the log for the given bundle. If no such log exists, one is created.
+	 *
+	 * @param bundle the bundle whose log is returned
+	 * @return the log for the given bundle
+	 * @since 3.28
+	 */
+	public static ILog getLog(Bundle bundle) {
```

Comment:
The Log part seems to be duplicated, since one the usually calls `ILog.getLog(bundle)`.
What about just naming the method `get` or `of` so that one gets more speaking code like `ILog.of(bundle)`.

The same applies for the method below.

Reply:
Done